health/health.d/wmi.conf

```diff
@@ -120,14 +120,52 @@ component: Network
        to: sysadmin
 
 
+## Service
+
+ template: wmi_service_status
+       on: wmi.service_status
+    class: Errors
+     type: Windows
+component: Services
+       os: *
+    hosts: *
+ families: *
+   lookup: max -60s unaligned of error,stressed,nonrecover
+    units: failures
+    every: 1m
+     warn: $this >= 1
+    delay: down 10m multiplier 1.5 max 30m
+     info: service $label:service is not operating properly
+       to: sysadmin
+
+
+## TCP
+
+# template: wmi_tcp_connections_failure
+#       on: wmi.tcp_conns_failures
+#    class: Errors
+#     type: Windows
+#component: TCP
+#       os: *
+#    hosts: *
+# families: *
+#   lookup: sum -10m unaligned absolute match-names of tcp_conns_failures_ipv4,tcp_conns_failures_ipv6
```

Comment:
```suggestion
#   lookup: sum -10m unaligned of ipv4,ipv6
```

@thiagoftsm this alarm doesn't work because `tcp_conns_failures_ipv4,tcp_conns_failures_ipv6` match nothing. It means that you don't test alarms before submitting. Why?

Reply:
Done,